plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import os from 'os';
+import { resolve as resolvePath } from 'path';
+import fs from 'fs-extra';
+import mockFs from 'mock-fs';
+import { getVoidLogger, UrlReader } from '@backstage/backend-common';
+import { ScmIntegrations } from '@backstage/integration';
+import { PassThrough } from 'stream';
+import { fetchContents } from './helpers';
+import { ActionContext, TemplateAction } from '../../types';
+import { createFetchTemplateAction, FetchTemplateInput } from './template';
+
+jest.mock('./helpers', () => ({
+  fetchContents: jest.fn(),
+}));
+
+const aBinaryFile = fs.readFileSync(
+  resolvePath(
+    'src',
+    '../fixtures/test-nested-template/public/react-logo192.png',
```

Comment:
Is this gonna work on windows?

Reply:
i think resolve runs normalize too on the end result

Reply:
yeah https://nodejs.org/api/path.html#path_path_resolve_paths

> The resulting path is normalized and trailing slashes are removed unless the path is resolved to the root directory.

Reply:
Yep!